changelog/unreleased/kong/bump-atc-router-1.3.1.yml

```diff
@@ -0,0 +1,3 @@
+message: Bumped atc-router from 1.2.0 to 1.3.1
```

Comment:
```suggestion
message: Bumped `atc-router` from `1.2.0` to `1.3.1`
```

Reply:
We don't use back quote for version bump in change log.